subsys/partition_manager/Kconfig

```diff
@@ -119,6 +119,17 @@ config PM_SINGLE_IMAGE
 
 DT_CHOSEN_EXT_FLASH:= nordic,pm-ext-flash
 
+config PM_EXTERNAL_FLASH_HAS_DRIVER
```

Comment:
Should we make `PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY` depend on this being enabled?

Couldn't that further assist users by failing in build time instead of run time?

Reply:
Good point.